src/dashboard/src/pages/JobsV2/AllJobs.tsx

```diff
@@ -150,7 +150,7 @@ const AllJobs: FunctionComponent = () => {
   return (
     <>
       { title && <Helmet title={title}/> }
-      { [ 'Unapproved', 'Pending', 'Running', 'Paused' ].map(
+      { [ 'Running', 'Unapproved', 'Pending', 'Paused' ].map(
```

Comment:
Running, Pending, Unapproved, Paused.

Same for MyJobs.tsx

Reply:
done